misc/scripts/ucaps_fetch.py

```diff
@@ -18,13 +18,12 @@
 URL: Final[str] = "https://www.unicode.org/Public/16.0.0/ucd/UnicodeData.txt"
 
 
-lower_to_upper: List[Tuple[str, str]] = []
-upper_to_lower: List[Tuple[str, str]] = []
-
-
-def parse_unicode_data() -> None:
+def fetch_unicode_data() -> Tuple[List[Tuple[str, str]], List[Tuple[str, str]]]:
     lines: List[str] = [line.decode("utf-8") for line in urlopen(URL)]
 
+    lower_to_upper = []
+    upper_to_lower = []
```

Comment:
```suggestion
    lower_to_upper: List[Tuple[str, str]] = []
    upper_to_lower: List[Tuple[str, str]] = []
```

Explicit type hints, in the vein of what was suggested to me in https://github.com/godotengine/godot/pull/101878#pullrequestreview-2570187303.

Reply:
Good idea, will do that on the next rebase!